.travis.yml

```diff
@@ -30,7 +32,9 @@ branches:
 # Clones WordPress and configures our testing environment.
 before_script:
     - export SLUG=$(basename $(pwd))
-    - svn co --quiet http://develop.svn.wordpress.org/tags/$WP_VERSION $WP_CORE_DIR
+    - export WP_TESTS_DIR=/tmp/wordpress/tests/phpunit
+    - export WP_CORE_DIR=/tmp/wordpress/
+    - git clone --depth=1 --branch="$WP_VERSION" git://develop.git.wordpress.org/ $WP_CORE_DIR
```

Comment:
> Is that the case?

Yes it is, wordpress.org runs `master`, if `master` goes 💥 , many people know about it very quickly 😏 

It also does work as a great way to find any issues in the upcoming release, before it is released, I'd suggest keeping it in personally 😄